### DISCUSSION TITLE
Does anyone know where I can find adapter reservation updates?

### DISCUSSION THREAD

**Discussion by apbarratt** [john30/ebusd#944] [Q&A]
My eBus Adapter reservation states that it is "Waiting for production, estimated for 06/2023".

I'm wondering if this is still the case?  It'd be nice to find a page of manufacturing status updates, hell, even just a Mastodon or twitter feed would be nice.  Just so I know if it's coming along nicely or if there have been delays etc, but I can't seem to find such a page linked to on the website :)

**Comment by mattsgreen** [comment]
Mine seems to have updated now, to say it's ready but will ship after Holiday in August ☹️

**Comment by apbarratt** [comment]
Eugh, that’s annoying, if they’d known there was a holiday at the time coming up they could have made that more clear :(On 3 Jul 2023, at 11:06, mattsgreen ***@***.***> wrote:﻿
Mine seems to have updated now, to say it's ready but will ship after Holiday in August ☹️
